src/main/scala/wdl4s/wom/graph/hmm.wdl

```diff
@@ -0,0 +1,11 @@
+workflow foo {
```

Comment:
this looks a lot like a "I'll just test some WDLing in intellij for the syntax highlighting" which accidentally got committed